archeryutils/rounds.py

```diff
@@ -19,24 +19,15 @@ class Pass:
     ----------
     n_arrows : int
         number of arrows in this pass.
-    scoring_system : {\
-        ``"5_zone"`` ``"10_zone"`` ``"10_zone_compound"`` ``"10_zone_6_ring"``\
-        ``"10_zone_5_ring"`` ``"10_zone_5_ring_compound"`` ``"WA_field"`` ``"IFAA_field"``\
-        ``"IFAA_field_expert"`` ``"Beiter_hit_miss"`` ``"Worcester"`` ``"Worcester_2_ring"``}
-        target face/scoring system type. Must be one of the supported values.
-    diameter : float or tuple of float, str
-        face diameter in [centimetres].
-    distance : float or tuple of float, str
-        linear distance from archer to target in [metres].
-    indoor : bool, default=False
-        is round indoors for arrow diameter purposes?
+    target : Target
+        A Target object.
 
     Attributes
     ----------
     n_arrows : int
         number of arrows in this pass.
     target : Target
-        A Target object defined using input parameters.
+        A Target object.
 
     Examples
```

Comment:
The examples here need updating to use the new `__init__` method (rather than replacing please move them to `at_target` which currently lacks examples).

ll. 184 (example for `Round`) can also be updated to use the new notation (worth a check over to see if I missed any others anywhere...).